sigstore/_internal/fulcio/client.py

```diff
@@ -83,71 +84,65 @@ class FulcioSCTError(Exception):
     pass
 
 
-class FulcioSignedCertificateTimestamp(SignedCertificateTimestamp):
-    def __init__(self, b64_encoded_sct: str):
-        self.struct = json.loads(base64.b64decode(b64_encoded_sct).decode())
+class FulcioSCT(BaseModel):
+    version: Version = Field(..., alias="sct_version")
+    log_id: bytes = Field(..., alias="id")
+    raw_timestamp: int = Field(..., alias="timestamp")
```

Comment:
N.B.: We have to keep this timestamp as a raw integer, since round-tripping it through `datetime.datetime` can result in us losing millisecond fidelity.